include/circt/Dialect/Arc/ArcOps.td

```diff
@@ -672,11 +679,15 @@ def ModelOp : ArcOp<"model", [
   }];
   let arguments = (ins SymbolNameAttr:$sym_name,
                        TypeAttrOf<ModuleType>:$io,
-                       OptionalAttr<FlatSymbolRefAttr>:$initialFn);
+                       OptionalAttr<FlatSymbolRefAttr>:$initialFn,
+                       OptionalAttr<FlatSymbolRefAttr>:$finalFn);
```

Comment:
The initial symbol has a verifier. Should we do the same for the final symbol? (and also add a test for it)